examples/carousel/esp-idf/s2-kaluga-kit/main/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: MIT
+
+idf_component_register(
+    SRCS "../../main.cpp"
+    INCLUDE_DIRS "."
+    REQUIRES slint
+)
+
+slint_target_sources(${COMPONENT_LIB} ../../../ui/carousel_demo.slint)
+target_link_options(${COMPONENT_LIB} PUBLIC -Wl,--allow-multiple-definition)
```

Comment:
I wonder if this shouldn't be in our slint component

Reply:
It turns out that they carousel example doesn't need them. I've removed them.

Reply:
the printerdemo might not need them either then.  This was possibly just needed when i was trying to use the platform from mcu-board-support for A.